lib/entity/event/save.js

```diff
@@ -0,0 +1,5 @@
+module.exports = function (args, done) {
+  var seneca = this;
+  var event = args.event;
+  seneca.make$('cd/events').save$(event);
```

Comment:
Missing call to done.

Reply:
This should have been caught by tests if i've handled doneSpy.to.have.been.calledOnce;